web/app/project/project.component.html

```diff
@@ -27,8 +27,11 @@
         <!-- Lane Column -->
         <ng-container matColumnDef="sha">
           <mat-header-cell *matHeaderCellDef>Commit Sha</mat-header-cell>
-          <!-- TODO: link this to the GitHub commit -->
-          <mat-cell *matCellDef="let build">{{build.shortSha}}</mat-cell>
+          <mat-cell *matCellDef="let build">
+            <a href="{{build.linkToSha}}">
```

Comment:
`[href]="build.linkToSha"`

Reply:
actually i think it's `[attr.href]`

Reply:
Do you think we should open this in a new tab?

Reply:
let's be greedy, new tab incoming 😃

Reply:
![](https://media1.tenor.com/images/4d8fff9c7ec293c2e76a941f3c46e66e/tenor.gif?itemid=5159819)